docs/json_schemas/kratos_external_idp/v0/requirer.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "RequirerSchema",
+  "description": "Requirer schema for KratosExternalIdp.",
```

Comment:
Could you expand this, explain who this is for, and when you might use it.

Reply:
I tried to add some more info. Tbh this relation interface will only be used by the Kratos charm and the Kratos integrator, no one else should ever use this.